nima_core/federation/hub_entity_resolver.py

```
@@ -0,0 +1,488 @@
+                        alias_name=alias_name,
+                        alias_type=alias_type,
+                        canonical_name=canonical_name,
+                        canonical_type=canonical_type,
+                    )
+                    conn.commit()
+                    if not merged:
+                        return False
+                except Exception:
+                    conn.rollback()
+                    raise
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)